inference-engine/src/multi_device/multi_device_plugin.cpp

```diff
@@ -168,16 +205,39 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadExeNetworkImpl(c
     }
 
     auto fullConfig = mergeConfigs(_config, config);
+    // collect the settings that are applicable to the devices we are loading the network to
+    std::unordered_map<std::string, InferenceEngine::Parameter> multiNetworkConfig;
+    std::vector<DeviceInformation> metaDevices;
+    auto workMode = fullConfig.find(CONFIG_KEY_INTERNAL(WORK_MODE));
```

Comment:
maybe better to rename the var's name to the auto_mode ( in the next PR)

Reply:
yes, thanks maxim, will rename